services/common/clients/config/trusted_server_config_client.cc

```diff
@@ -109,7 +110,8 @@ absl::Status TrustedServersConfigClient::Init(
 
 bool TrustedServersConfigClient::HasParameter(
     absl::string_view name) const noexcept {
-  return config_entries_map_.contains(name);
+  return config_entries_map_.contains(name) &&
+         config_entries_map_.at(name) != kEmptyValue;
```

Comment:
This is a change included in the previous PR https://github.com/privacysandbox/bidding-auction-servers/pull/7.